common-lib/src/test/java/com/google/cloud/tools/intellij/BasePluginInfoServiceTest.java

```diff
@@ -31,10 +31,10 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mock;
-import org.mockito.runners.MockitoJUnitRunner;
+import org.mockito.junit.MockitoJUnitRunner;
 
 /** Tests for {@link BasePluginInfoService}. */
-@RunWith(MockitoJUnitRunner.class)
+@RunWith(MockitoJUnitRunner.Silent.class)
```

Comment:
so this is just happening in base test classes where the mock is intended to be used by sub classes? I wonder if there's a way to mark it as "silent" at a more granular level rather than replacing the runner for the whole class.